composer.json

```
@@ -14,10 +14,10 @@
     "require": {
         "php": ">=8.0",
         "utopia-php/framework": "0.*.*",
+        "ext-pdo": "*",
         "utopia-php/cache": "0.8.*"
```

```suggestion
        "php": ">=8.0",
        "utopia-php/cache": "0.8.*",
        "utopia-php/framework": "0.*.*",
        "ext-pdo": "*"
```

Let's keep the list organized.